### ISSUE TITLE
plugin(moment): Module "moment" has no default export.

### ISSUE THREAD

**Issue by Nols1000** [fullcalendar/fullcalendar#4580]
### Bug Report
Using moment 2.24.0 I get a typescript error:
```
ERROR in node_modules/@fullcalendar/moment/main.d.ts(2,12): error TS1192: Module '"***/node_modules/moment/moment"' has no default export.
```

**Comment by acerix** [comment]
Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by Nols1000** [comment]
Repository to reproduce the issue:
https://github.com/Nols1000/fullcalendar-moment-reproduce

**Comment by ganySA** [comment]
I can confirm this issue.

**Comment by acerix** [comment]
Thanks for the demo, confirmed, I get the same error.

**Comment by Nols1000** [comment]
I will try that tomorrow and report back.

**Comment by Nols1000** [comment]
@arshaw It works with `allowSyntheticDefaultImports` set to `true`.

**Comment by MVMurrell** [comment]
@arshaw just downloaded fc 8.6.19. Has the below problem been added to the milestone?

declare module "@fullcalendar/moment" {
    import * as moment from "moment";
    import { Calendar, Duration } from "@fullcalendar/core";
    export function toMoment(date: Date, calendar: Calendar): moment.Moment;
    export function toDuration(fcDuration: Duration): moment.Duration;
    const _default_10: import("@fullcalendar/core/plugin-system").PluginDef;
    export default _default_10;
}

The 1st line needs the '* as' in import moment.

**Comment by arshaw** [comment]
This was fixed in 4.3.0 though please start using 4.3.1
Releases: https://github.com/fullcalendar/fullcalendar/releases
